code-review.qmd

```diff
@@ -46,7 +46,9 @@ git push origin review
 
 The pull request can now be made from the `review` branch to the `v_020` branch and will provide the difference between versions.
 
-This process is similar to the [release comparison feature provided by Github](https://docs.github.com/en/repositories/releasing-projects-on-github/comparing-releases), but that feature does not allow commenting and suggesting like the pull request interface.
+This process is similar to the [release comparison feature provided by GitHub](https://docs.github.com/en/repositories/releasing-projects-on-github/comparing-releases), but that feature does not allow commenting and suggesting like the pull request interface.
+
+Now that the partial review pull request is set up, see the [Addressing package reviews section](https://epiverse-trace.github.io/blueprints/code-review.html#addressing-package-reviews) for ways to incorporate changes from reviewer's comments and complete the code review.
```

Comment:
thank you for adding this line! now the process and next step is more clear!

Reply:
Thanks for confirming.